src/main/java/games/strategy/net/ServerMessenger.java

```diff
@@ -339,7 +339,7 @@ private void bareBonesSendChatMessage(final String message, final INode to) {
       rn = new RemoteName(ChatController.getChatChannelName("_LOBBY_CHAT"), IChatChannel.class);
     } else {
       rn = new RemoteName(
-          ChatController.getChatChannelName("games.strategy.engine.framework.ui.ServerStartup.CHAT_NAME"),
+          ChatController.getChatChannelName("games.strategy.engine.framework.uiComponent.ServerStartup.CHAT_NAME"),
```

Comment:
Was this intentional?

Reply:
Bump

Reply:
fixed in: 5f874b15cf8af4641c58db645fd130701cea49be